src/nibiru/index.ts

```diff
@@ -0,0 +1,73 @@
+import type { LocalConfig } from '@/stores';
+
+const PLAYGROUND_NETWORKS = 'https://networks.play.nibiru.fi/ping-pub';
+const DEV_NETWORKS = 'https://networks.testnet.nibiru.fi/ping-pub';
+const ITN_NETWORKS = 'https://networks.itn.nibiru.fi/ping-pub';
+
+export const getItn = async (): Promise<LocalConfig[] | undefined> => {
+  try {
+    const testnets = await fetch(ITN_NETWORKS).then((response) =>
+      response.json()
+    );
+    testnets.forEach((_: any, i: string | number) => {
```

Comment:
Could be .map

Reply:
Same for other .forEach

Reply:
I get this warning from sonar 
`Consider using "forEach" instead of "map" as its return value is not being used here.sonarlint(typescript:S2201)`